tests/python/contrib/test_ethosu/test_networks.py

```diff
@@ -20,7 +20,7 @@
 pytest.importorskip("ethosu.vela")
 
 import numpy as np
-
+import tvm
```

Comment:
on the 27th line the same import remained

Reply:
removed